README.md

```diff
@@ -151,7 +151,14 @@ Some of the inputs for this module are tags. All infrastructure resources must b
 
 You should use your namespace variables to populate these. See the [Usage](#usage) section for more information.
 
+## Team name caveat
+
+This module utilises your environemnt `team_name` variable in the naming of your SQS topic, in the format `cloud-platform-<var.team_name>-<var.environment_name><ar.sqs_name>`. This historically introduced an issue whereby a team name change would result in Terraform forcefully replacing the SQS topic. To get around this issue, a [lifecycle](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle) `ignore_changes` block has been introduced, so that team name changes can be made without causing this issue.
```

Comment:
```suggestion
This module utilises your environment `team_name` variable in the naming of your SQS topic, in the format `<var.team_name>-<var.environment_name><var.sqs_name>`. This historically introduced an issue whereby a team name change would result in Terraform forcefully replacing the SQS topic. To get around this issue, a [lifecycle](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle) `ignore_changes` block has been introduced, so that team name changes can be made without causing this issue.
```